src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -93,11 +95,11 @@ class DefaultStorageKeyAllocator
     }
     const char * FabricKeyset(chip::FabricIndex fabric, uint16_t keyset) { return Format("f/%x/k/%x", fabric, keyset); }
 
-    const char * AttributeValue(const app::ConcreteAttributePath & aPath)
+    const char * AttributeValue(EndpointId endpointId, ClusterId clusterId, AttributeId attributeId)
```

Comment:
I guess this is OK, but does raise the question of whether the ConcreteAttributePath spec type should be in DataModelTypes somehow....